site_src/Assets/input.css

```
@@ -1349,14 +1394,109 @@ code, pre, kbd, .font-mono {
 }
+.tcg-side-close:hover { color: #fff; border-color: var(--accent); }
+.tcg-side-panel.in-modal .tcg-side-close { display: inline-flex; }
+
+/* The feed: action-log entries + chat bubbles interleaved, newest at the bottom. */
+.tcg-feed {
+  flex: 1 1 auto; overflow-y: auto; display: flex; flex-direction: column; gap: 0.18rem;
+  max-height: 300px; min-height: 0;
+  font-family: 'JetBrains Mono', monospace; font-size: 0.72rem; color: var(--fog-300);
+}
+/* Chat shows as a plain "name: message" line in the feed (not a bubble). */
+.tcg-chat-line { line-height: 1.4; margin: 0.06rem 0; word-break: break-word; overflow-wrap: anywhere; }
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)